XRTK-Core/Packages/com.xrtk.core/Services/SpatialAwarenessSystem/MixedRealitySpatialAwarenessSystem.cs

```diff
@@ -45,7 +45,7 @@ private GameObject CreateSpatialAwarenessParent
             get
             {
                 var spatialAwarenessSystemObject = new GameObject("Spatial Awareness System");
-                spatialAwarenessSystemObject.transform.parent = MixedRealityToolkit.CameraSystem?.CameraRig.PlayspaceTransform;
+                spatialAwarenessSystemObject.transform.SetParent(MixedRealityToolkit.CameraSystem?.MainCameraRig.PlayspaceTransform, false);
```

Comment:
We should check if the camera system is null here and fall back to `CameraCache.Main.parent`